src/main/scala/extractor/ExtractImplicits.scala

```diff
@@ -0,0 +1,72 @@
+package extractor
+
+import java.util.concurrent.ConcurrentLinkedQueue
+
+import extractor.Serializables.{Apply, DeclaredImplicit, FunApplyWithImplicitParam, ImplicitParam}
+import org.langmeta.inputs.Input
+
+import scala.meta._
+
+object ExtractImplicits {
```

Comment:
To me the whole object is essentially a function. This can be made explicit:
```scala
object ExtractImplicit extends (SemanticCtx => Result) { ... }
```